all-clouds/all-clouds-explanation/cloud-init-metapackages.rst

```diff
@@ -0,0 +1,21 @@
+Cloud-init metapackages
+=======================
+
+Starting in Ubuntu Oracular (24.10), `cloud-init <https://launchpad.net/ubuntu/+source/cloud-init>`_
+added cloud-specific metapackages to better express cloud-specific dependencies and reduce image size.
+
+* ``cloud-init-base`` contains the cloud-init code with the minimal set of dependencies
+
+* ``cloud-init`` is a metapackage that retains every previous dependency
+
+* ``cloud-init-cloud-sigma`` and ``cloud-init-smart-os`` are metapackages with an
+  additional dependency on python3-serial
+
+If you are an image builder:
+
+* Install ``cloud-init``, containing all dependencies, if the target platform is unknown
+
+* Install a cloud-specific metapackage if the target platform requires one
```

Comment:
Move this point to the end of the list (after 'Install ``cloud-init-base`` ... )
```suggestion
* Install a cloud-specific metapackage if the target platform has some specific dependency requirements. 
```

Reply:
Done.